src/helpers/userHelper.js

```diff
@@ -984,7 +1007,7 @@ const userHelper = function () {
         const user = users[i];
         const { endDate } = user;
         endDate.setHours(endDate.getHours() + 7);
-        if (moment().isAfter(moment(endDate))) {
+        if (moment().isAfter(moment(endDate).add(1, 'days'))) {
```

Comment:
Hi Yan, everything else looks good. Just feel a little confused about these lines.

For now, users will be deactivated after the endDate right? For example, a user's end date is Dec-10 and this account will turn inactive on Dec-11. 
From my understanding, `isAfter` has already checked if today's date is after the end date. If you add 1 more day on top of the end date here, the above example will not be true on Dec-11, it will be true on Dec-12. Because Dec-11 is ****NOT** after** endDate+1day which is also Dec-11.
Correct me if I was wrong!

Reply:
Thanks, Yan, that makes a lot more sense. Good Job 👍